ons_alpha/jinja2/templates/base.html

```diff
@@ -28,30 +28,30 @@
                 }
             ]
         }
-        },
-        "footer": {
-            "rows": [
-                {
-                    "itemsList": [
-                        {
-                            "text": 'Contact us',
-                            "url": '#0'
-                        },
-                        {
-                            "text": 'Cookies and privacy',
-                            "url": '#0'
-                        }
-                    ]
-                }
-            ],
-            "OGLLink": {
-                "pre": 'All content is available under the',
-                "link": 'Open Government Licence v3.0',
-                "url": 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/',
-                "post": ', except where otherwise stated'
+    },
```

Comment:
There's no change here, just fixes the whitespace so the brackets match and it's easier to follow